ibis/backends/dask/execution/numeric.py

```diff
@@ -0,0 +1,132 @@
+import collections
+import decimal
+import functools
+import numbers
+
+import dask.dataframe as dd
+import dask.dataframe.groupby as ddgb
+import numpy as np
+
+import ibis.common.exceptions as com
+import ibis.expr.operations as ops
+from ibis.backends.pandas.core import numeric_types
+from ibis.backends.pandas.execution.generic import execute_node
+
+
+# TODO - aggregations - #2553
+@execute_node.register(ops.Arbitrary, ddgb.SeriesGroupBy, type(None))
+def execute_arbitrary_series_groupby(op, data, _, aggcontext=None, **kwargs):
+    how = op.how
+    if how is None:
+        how = 'first'
+
+    if how not in {'first', 'last'}:
+        raise com.OperationNotDefinedError(
+            'Arbitrary {!r} is not supported'.format(how)
+        )
+    return aggcontext.agg(data, how)
+
+
+@execute_node.register(ops.Negate, dd.Series)
+def execute_series_negate(op, data, **kwargs):
+    return data.mul(-1)
+
+
+# TODO - grouping - #2553
+@execute_node.register(ops.Negate, ddgb.SeriesGroupBy)
+def execute_series_group_by_negate(op, data, **kwargs):
+    return execute_series_negate(op, data.obj, **kwargs).groupby(
+        data.grouper.groupings
+    )
+
+
+def call_numpy_ufunc(func, op, data, **kwargs):
+    if data.dtype == np.dtype(np.object_):
+        return data.apply(
+            functools.partial(execute_node, op, **kwargs),
+            meta=(data.name, "object"),
+        )
+    return func(data)
+
+
+@execute_node.register(ops.UnaryOp, dd.Series)
+def execute_series_unary_op(op, data, **kwargs):
+    function = getattr(np, type(op).__name__.lower())
+    return call_numpy_ufunc(function, op, data, **kwargs)
+
+
+@execute_node.register((ops.Ceil, ops.Floor), dd.Series)
+def execute_series_ceil(op, data, **kwargs):
+    return_type = np.object_ if data.dtype == np.object_ else np.int64
+    func = getattr(np, type(op).__name__.lower())
+    return call_numpy_ufunc(func, op, data, **kwargs).astype(return_type)
+
+
+def vectorize_object(op, arg, *args, **kwargs):
+    # TODO - this works for now, but I think we can do something much better
```

Comment:
yeah this almost doesn't even matter. its going the op on object dtype with numeric data; this is for numpy compat and likely never hit in any import case.